### PR TITLE
Config for Malyan M150

### PR DESCRIPTION
**Pull request by guntherv73**
This is a request to add an example config for the HobbyKing Maylan M150 config.
the files are up to date with current MarlinFirmware:RCBugFix

### CODE REVIEW

File: Marlin/example_configurations/M150/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Configuration for Maylyan M150 hobbyking printer
```
Comment (thinkyhead) [inline Marlin/example_configurations/M150/README.md:1]: "Malyan"

**Comment by bionicbone** [comment]
@thinkyhead I have some concerns about this pull. 

To get the Malyan M150 to compile and work correctly there are some hard coded and dirty fixes to core code that are required along with the items in this pull request. 

I have done many print tests with these fixes in place, there are no issues but I would suggest they may not be the preferred way of changing the master code :)

1. The pins are change by hard code in pins_SANGUINOLOLU_11.h
2. Some code is commented out in ultralcd_impl_DOGM.h
3. Some changes to code in ultralcd_st7920_u8glib_rrd.h

For  more details scroll to the bottom of this commit
https://github.com/bionicbone/Marlin/commit/728c871f3f6f6ef9a2e9c7c60ef0dfaeec5699f8
and see this one
https://github.com/bionicbone/Marlin/commit/71472178ea79bc1a39b36dc64ea4cfae5314ae43

Maybe some previous updates have broken the Melzi compile?

Either way I thought you'd want to know this when considering to pull this to the master code.

Kev

**Comment by guntherv73** [comment]
thank you Kev, not sure how i missed that :( 
Then the config is indeed useless, maybe document in the readme the changes ?
I think the effort to add it to the master files is not worth it seen the M150 seams EOL.
If it's deamed not usefull to pull then i'll doc it anyway on my fork for me.

**Comment by thinkyhead** [comment]
> I have done many print tests with these fixes in place, there are no issues but I would suggest they may not be the preferred way of changing the master code :)

Yes, it's definitely best if all such changes are made conditional. I wonder if all machines with custom boot screens should be skipping the code which is being commented out for Malyan. If so, then the condition would just be `#if ENABLED(SHOW_CUSTOM_BOOTSCREEN)` to disable that code.

**Comment by bionicbone** [comment]
I'm not close enough to this project to know the answer to that, however it should be easy enough to check.

I'll have a further look into it this weekend and update to see if I can find a belts and braces approach for Malyan users without compromising the master code.

**Comment by thinkyhead** [comment]
Updated and ready to merge at #6931.

**Pull request closed without merging**